src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -1177,11 +1249,48 @@ class NodeConfig(object):
                             convert_size(entries[3] + entries[4], 'kb')
                     elif entries[2] == "HugePages_Total:":
                         numa_nodes[num]['HugePages_Total'] = entries[3]
+        # Spread memory resources across NUMA nodes
+        if self.cfg['vnode_per_numa_node'] and len(numa_nodes) > 0:
```

Comment:
Since numa_nodes is a list, you can just check `if numa_nodes` instead of its length.

Reply:
Since we use len(numa_nodes) as a denominator we must ensure it is not zero. There was no sense recalculating this value in the block that followed, so assigned len(num_nodes) to a variable.